scripts/spawner.gd

```diff
@@ -3,27 +3,34 @@ extends Node2D
 var period = 2
 var elapsed = 0
 var spawnarea = position
+@export var enemy_target : Vector2 = Vector2(0,0)
+@export var concurrent_spawns : int = 10
+
+var spawned = []
+
+func n_can_spawn() -> int:
```

Comment:
Please rename to something clearer (like: get_can_spawn_amount)

Reply:
Also, can we store this value in a constructor or something? There's no need to compute this over and over every time we spawn

Reply:
I made it a function so that we could add different logic to check how many enemies we can spawn that can depend on ome other things but i can also store it in a variable